app/controllers/miq_ae_class_controller.rb

```diff
@@ -2453,6 +2453,10 @@ def git_refresh
     session[:changed] = true
 
     git_repo = MiqAeDomain.find(params[:id]).git_repository
+
+    git_based_domain_import_service.refresh(git_repo.id)
```

Comment:
@gmcculloug 
The refresh is returning true or false via the task_results to indicate if the refresh succeeded